src/Core/Extensions/GraphQLBuilderCoreExtensions.cs

```diff
@@ -31,11 +31,23 @@ public InternalErrorInfoProvider(IOptions<ErrorInfoProviderOptions> options)
         /// <param name="configureOptions">Action to configure the <see cref="ErrorInfoProviderOptions"/>.</param>
         /// <returns>Reference to <paramref name="builder"/>.</returns>
         public static IGraphQLBuilder AddErrorInfoProvider(this IGraphQLBuilder builder, Action<ErrorInfoProviderOptions> configureOptions)
+            => AddErrorInfoProvider<InternalErrorInfoProvider>(builder, configureOptions);
+
+        /// <summary>
+        /// Provides the ability to configure <see cref="ErrorInfoProviderOptions"/> for the specified <typeparamref name="TErrorInfoProvider"/>.
+        /// Also provides integration with Microsoft.Extensions.Options so the caller may use services.Configure{ErrorInfoProviderOptions}(...)
```

Comment:
It is not true. Let's make `InternalErrorInfoProvider` public and rename it to `DefaultErrorInfoProvider`. Next change constraint to  `where TErrorInfoProvider : class, DefaultErrorInfoProvider`.